app/models/energy/ScheduleFile.py

```diff
@@ -0,0 +1,59 @@
+"""Schedule File Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType
+
+
+class ScheduleFile(BaseModel):
+    """A list of values used to create a .csv file for the entire year or a period of
+  analysis within the year."""
+
+    type: Enum('ScheduleFile', {'type': 'ScheduleFile'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    start_day_of_year: int = Schema(
+        ...,
+        ge=1,
+        le=366,
+        description='Used to define the start day for the analysis period.'
+    )
+
+    values: List[int] = Schema(
```

Comment:
@chriswmackey  Should I add the `Interval Length`  and `value` instead of a list of values for `ScheduleFixedInterval`?